CHANGELOG.md

```diff
@@ -1,5 +1,26 @@
 # Changelog
 
+## RDMO 2.0.0 (Sep 11, 2023)
+
+* Refactor data model of questions app:
+  * Introduce Pages model, replacing question sets which are not nested
+  * Use m2m relations instead of foreign keys, e.g. one catalog has now many sections,
+    but one section can be also part of many catalogs
+  * Refactor import and update elements import format, but keep old format working
+  * Rename key to uri_path in all element models, but Attribute
+* Add new React/Redux-based management interface, which replaces the 6 old interfaces:
+  * Add table-like views for all elements, with locking, availability and export buttons
+  * Add nested views for catalogs, sections, pages and question sets
+  * Improve edit views for elements, remove modals
+* Add Site-based permissions for all elements and new editor and reviewer roles
+* Add set_collection to Value model to store if this value was part of a set
+* Add JSON project export and refactor CSV exports
+* Add refresh to option set providers
+* Don't open XML exports in browser, except when EXPORT_CONTENT_DISPOSITION = None
+* Fix new Shibboleth setup
+* Refactor packaging, add pyproject.toml, add pre-commit-config, overhaul testing and CI
```

Comment:
Maybe add "Update Python dependencies"? Earlier changelog entries have such points.

Updated so far are the following:
django
django-compressor
pypandoc

pytest
pytest.cov
pytest-django